lib/ClangImporter/SwiftLookupTable.h

```diff
@@ -153,7 +249,7 @@ const uint16_t SWIFT_LOOKUP_TABLE_VERSION_MAJOR = 1;
 /// Lookup table minor version number.
 ///
 /// When the format changes IN ANY WAY, this number should be incremented.
-const uint16_t SWIFT_LOOKUP_TABLE_VERSION_MINOR = 14; // Swift 2 names
+const uint16_t SWIFT_LOOKUP_TABLE_VERSION_MINOR = 15; // Special names
```

Comment:
Specifically, unlike swiftmodules, this table format is never distributed, just embedded into pcm files.